fnl/tangerine/utils/path.fnl

```diff
@@ -36,7 +49,7 @@
         path (path:gsub (.. "%." ext1 "$") (.. "." ext2))]
        (if (path:find from)
            (path:gsub from to)
```

Comment:
I think there could be a bug in line 50 and 51, you should use `p.match` instead of `path:find` and `p.gsub` instead of `path:gsub` just to be sure.